src/StreamJsonRpc.Tests/JsonRpcTests.cs

```diff
@@ -1635,11 +1505,13 @@ public int PlusTwo(int arg)
         }
     }
 
+    [DataContract]
     public class Foo
     {
-        [JsonProperty(Required = Required.Always)]
+        [DataMember(Order = 0, IsRequired = true)]
```

Comment:
Is ordering required for MessagePack?

Reply:
Yes, in its optimal mode. It *can* be configured to use named properties but it makes the binary format much bigger, so it isn't the recommended approach.